compiler-core/templates/gleam@@compile.erl

```diff
@@ -5,7 +5,18 @@
 
 -record(arguments, {lib = "./", out = "./", modules = []}).
 
-main(Args) ->
+main(_) -> main().
+
+main() ->
```

Comment:
How about a more clear name such as "compile_package_loop"? I didn't realise this ran more than once at first as it's called "main"